events_structs.go

```diff
@@ -289,7 +296,17 @@ func (ev *EventRunType) String() string {
 	if ev.TypeData.Detached {
 		detached = "detached "
 	}
-	return fmt.Sprintf("%sone-off container with command '%s'", ev.TypeData.Command, detached)
+
+	if ev.isEventRunFromOperator() {
+		// The command executed is not available to end user if it's executed by a Scalingo operator
+		return fmt.Sprintf("%sone-off container for maintenance/support purposes", detached)
+	}
+
+	return fmt.Sprintf("%sone-off container with command '%s'", detached, ev.TypeData.Command)
+}
+
+func (ev *EventRunType) isEventRunFromOperator() bool {
```

Comment:
Isn't it more if the comande is run by the generic user?

Reply:
What do you mean by "generic user"?